mix.exs

```diff
@@ -43,6 +43,7 @@ defmodule Explorer.MixProject do
       {:table_rex, "~> 3.1.1 or ~> 4.0.0"},
       {:castore, "~> 1.0", optional: true},
       {:adbc, "~> 0.1", optional: true},
+      {:tz, "~> 0.26"},
```

Comment:
This should be a `only: :test` dependency. We should not enforce a TZ database onto users.

Reply:
Yeah I agree

Reply:
Addressed by: [306c789](https://github.com/elixir-explorer/explorer/pull/903/commits/306c7895e61470ef2e10927f2875a1de8212bcd2)